Tasks/UsePythonV1/proxyutil.ts

```diff
@@ -0,0 +1,17 @@
+import * as task from 'azure-pipelines-task-lib/task';
+import * as qs from 'querystring';
+
+export function setProxy(proxyCfg: task.ProxyConfiguration) {
+    const proxy = new URL(proxyCfg.proxyUrl);
+    const proxyEnvVarName: string = proxy.protocol === 'https:'? "https_proxy" : "http_proxy";
```

Comment:
I'd just let TS infer the type here.  It will infer `"https_proxy" | "http_proxy"`, which is a stricter type.